src/schema/v2/show_event.ts

```diff
@@ -24,8 +24,10 @@ const ShowEventType = new GraphQLObjectType<any, ResolverContext>({
     dateTimeRange: {
       type: GraphQLString,
       description: "A formatted description of the dates with hours",
-      resolve: ({ start_at, end_at }) =>
-        dateTimeRange(start_at, end_at, "UTC", true),
+      resolve: ({ start_at, end_at }, { timezone }, { defaultTimezone }) => {
```

Comment:
What's the `timezone` argument here? (sorry just back from 🍹 and catching up on GH). This might be missing something that lets `timezone` be passed in as an argument to the field (but that might not even be needed at all, vs. just using `defaultTimezone`).

That's something you get with the `date` helper (https://github.com/artsy/metaphysics/blob/e8b80747acea25b8dcdc31ffe9c86f0e59f53c47/src/schema/v2/fields/date.ts#L35), but otherwise would need to include that if wanting to use that here. However, if you only plan on using this via `defaultTimezone` and _not_ querying it like `dateTimeRange(timezone: "EST")`in the GraphQL query, you can drop it completely.

Reply:
Hrm but isn't that timezone value populated by the header?